rqt_py_common/src/rqt_py_common/message_tree_model.py

```diff
@@ -62,6 +62,18 @@ def _get_toplevel_items(self, index_list):
     def _get_data_items_for_path(self, slot_name, slot_type_name, slot_path, **kwargs):
         return (QStandardItem(slot_name), QStandardItem(slot_type_name), QStandardItem(slot_path))
 
+    def _get_property_name(self, slot_name, message_instance):
+        # slots start with an underscore
+        property_name = slot_name[1:]
```

Comment:
You shouldn't hardcode this without first checking. First check the length >1 and that the first character is an underscore.